packages/aurora-editor/modifiers/canSubmit.js

```diff
@@ -47,6 +47,7 @@ function canSubmit(Editor) {
     render() {
       return (
         <Editor
+          placeholder="Type here to search or to create a note"
```

Comment:
This is fine for now, but we should look into putting all of this user-facing text in a single file eventually. That'll also make it easier to translate the app into other languages.

Reply:
When Jira is back up, I'll add this to the backlog. Maybe an `aurora-resources`? Or `aurora-strings`? Like how Android does it?